src/components/Interval.tsx

```diff
@@ -1,5 +1,6 @@
 import { Dimensions } from 'react-native'
-import { Box, Text } from '@components'
+import { Box } from './Box'
```

Comment:
These changes were to resolve circular deps

Reply:
I think a more "proper" fix for these issues would be to update the type imports to use `import type` instead of just `import`. This is just a guess, but I think that's where the circular imports are coming from.